Documentation/manpages/sdk/dotnet-add-package.1

```diff
@@ -1,7 +1,16 @@
 .\" Automatically generated by Pandoc 2.1.3
 .\"
-.TH "dotnet add package command \- .NET Core CLI" "1" "" "" ".NET Core"
+.TH "" "1" "" "" ".NET Core"
 .hy
+.PP
+   *   *   *   *   *
+.PP
+title: dotnet add package command \- .NET Core CLI description: The \[aq]dotnet add package\[aq] command provides a convenient option to add a NuGet package reference to a project.
+author: mairaw ms.author: mairaw ms.date: 08/11/2017 ms.topic: article ms.prod: .net\-core ms.technology: dotnet\-cli ms.workload:
+.IP \[bu] 2
+dotnetcore
+.PP
+   *   *   *   *   *
```

Comment:
Is the addition of `title: ...` and `author: ...` here caused by the change in `man` to `gfm` in the pandoc command? It doesn't render too well on my machine:

![image](https://user-images.githubusercontent.com/174451/38839818-75e74c76-41aa-11e8-8dfc-895dc99027a6.png)

Reply:
It was detecting the markdown as command mark, instead of gfm. So I have passed `-f` (--from gfm) argument. We need to modify python script. Can you try it on Debian? I have updated the readme with copy/pastable steps.

Reply:
Turned out it dotnet/docs is using docfx instead of gfm, which doesn't support author properly (missing from documentation). I have opened https://github.com/jgm/pandoc/issues/4557 for docfx support and removed `-f gfm`.